### DISCUSSION TITLE
need ide extension

### DISCUSSION THREAD

**Discussion by yebei199** [terrastruct/d2#2217] [Feature Requests]
like vsc d2 extension

**Comment by stereobooster** [comment]
This one https://marketplace.visualstudio.com/items?itemName=terrastruct.d2?

**Reply by yebei199** [reply]
> This one [marketplace.visualstudio.com/items?itemName=terrastruct.d2](https://marketplace.visualstudio.com/items?itemName=terrastruct.d2)?这是[marketplace.visualstudio.com/items?itemName=terrastruct.d2](https://marketplace.visualstudio.com/items?itemName=terrastruct.d2)吗？

this is vsc , not ide such as webstorm,rider
